src/phrases/phrases-en.ts

```diff
@@ -1247,6 +1247,7 @@ const phrases = {
     confirm: 'Confirm',
     loginAgain: 'Login again',
   },
+  environment: { production: 'production', staging: 'staging', test: 'test' },
```

Comment:
Tenker at vi ikkje treng navn på prod. Det er viktigst at en blir minna på at en er på test.
```suggestion
  environment: { production: '', staging: 'Staging', test: 'Test' },
```